.github/workflows/main.yml

```diff
@@ -138,6 +138,9 @@ jobs:
     - run: cargo check -p wasmtime --no-default-features --features uffd
     - run: cargo check -p wasmtime --no-default-features --features cranelift
     - run: cargo check -p wasmtime --no-default-features --features cranelift,wat,async,cache
+    
+    # Check that benchmarks of the cranelift project build
+    - run: cargo bench -p cranelift-codegen
```

Comment:
`cargo check --benches` will check all benchmarks without running them.

Reply:
I see now, that I misunderstood the issue itself.
I will fix this up.

Would it also make sense to create another CI check, to verify that the benchmarks are running?
I am not sure, if there's an existing strategy (or desire to have one) to protect against regressions in the CLIF format and the way it's getting executed.